### ISSUE TITLE
Measure validation loss

### ISSUE THREAD

**Issue by UditGupta10** [thunlp/OpenKE#109]
In a typical ML setup we have the loss curves of training and validation to determine the optimum number of iterations to stop at. How to see the validation loss in this case. Here we get the metrics only at last, no way to verify if it underfits or overfits.
Please tell how to measure the validation loss as number of iterations increase.

**Comment by ShulinCao** [comment]
I have added the validation loss. Have a try!

**Comment by UditGupta10** [comment]
I do not see the modification in config.py, kindly tell which file has been modified.

**Comment by UditGupta10** [comment]
@ShulinCao Could you please link to the location in code where validation loss part is added, I am unable to find it.

**Comment by ShulinCao** [comment]
https://github.com/thunlp/OpenKE/blob/0a55399b3e800bc779582c4784cac96f00230fd8/base/Valid.h#L1
https://github.com/thunlp/OpenKE/blob/0a55399b3e800bc779582c4784cac96f00230fd8/config/Config.py#L370
https://github.com/thunlp/OpenKE/blob/0a55399b3e800bc779582c4784cac96f00230fd8/config/Config.py#L406

**Comment by yib0liu** [comment]
Could you add the validation code to the new Pytorch version? Thanks!
